### PR TITLE
Add "GraphQL" prefix to GraphQL entities

### PR DESCRIPTION
**Pull request by Betree**
Resolve https://github.com/opencollective/opencollective/issues/6403#issuecomment-1423869843

Not covering Inputs and Collections since we don't have any risk of name conflicts right now, but can add that if others think it's important.

### CODE REVIEW

**Review by Loyalty84 — COMMENTED** [review]
GraphQLImageFileInfo,
  GraphQLGenericFileInfo,
  GraphQLIndividual,
  GraphQLMember,
  GraphQLMemberOf,
  GraphQLOrganization,
  GraphQLTransferWise,
  GraphQLVendor,
  GraphQLVirtualCard,
];

File: server/graphql/v2/types.js [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+  GraphQLIndividual,
+  GraphQLMember,
+  GraphQLMemberOf,
+  GraphQLOrganization,
+  GraphQLTransferWise,
+  GraphQLVendor,
+  GraphQLVirtualCard,
 ];
 
 export default types;
```
Comment (Loyalty84) [inline server/graphql/v2/types.js:40]: GraphQLImageFileInfo,
  GraphQLGenericFileInfo,
  GraphQLIndividual,
  GraphQLMember,
  GraphQLMemberOf,
  GraphQLOrganization,
  GraphQLTransferWise,
  GraphQLVendor,
  GraphQLVirtualCard,

Comment (Loyalty84) [reply server/graphql/v2/types.js:40]: Type

**Review by Loyalty84 — APPROVED** [review]
GraphQLImageFileInfo,
  GraphQLGenericFileInfo,
  GraphQLIndividual,
  GraphQLMember,
  GraphQLMemberOf,
  GraphQLOrganization,
  GraphQLTransferWise,
  GraphQLVendor,
  GraphQLVirtualCard,

**Comment by hdiniz** [comment]
I dont see any blockers. For consistency, I would consider also renaming the inputs and collections.

**Review by hdiniz — APPROVED** [review]

**Comment by Betree** [comment]
> I dont see any blockers. For consistency, I would consider also renaming the inputs and collections.

Makes sense. I'll iterate on this before merging.

**Review by znarf — APPROVED** [review]
Good for me!

**Comment by Betree** [comment]
For reference, I've tried to enforce this with an ESLint rule but haven't found an easy way. I've documented a new issue in https://github.com/typescript-eslint/typescript-eslint/issues/6854 to add a feature to the `naming-convention` rule that would let us apply specific rules on GraphQL types. Hopefully, we'll be able to enforce that in the future.

**Pull request merged**